scripts/globals/abilities/third_eye.lua

```diff
@@ -17,5 +17,9 @@ function onAbilityCheck(player, target, ability)
 end
 
 function onUseAbility(player, target, ability)
-    player:addStatusEffect(tpz.effect.THIRD_EYE, 0, 0, 30) --power keeps track of procs
+    if (player:hasStatusEffect(tpz.effect.COPY_IMAGE)) then
```

Comment:
outermost parenthesis are not required. you should use parenthesis on things that are functions like `addStatusEffect()` needs them, or to clarify order of operations like `if a and (b or c) and d then`

So here you would want
```lua
 if player:hasStatusEffect(tpz.effect.COPY_IMAGE) then
```